CMakeLists.txt

```diff
@@ -68,6 +68,9 @@ include(ROCMCreatePackage)
 include(CheckCXXCompilerFlag)
 include(ROCMHeaderWrapper)
 
+# Build library with Beta APIs
+add_definitions("-DMIOPEN_BETA_API=1")
+
```

Comment:
[Recommendation] Remove all conditionals like 
```C++
#ifdef MIOPEN_BETA_API
...
#endif
```
from the library sources EXCEPT miopen.h.

But this can be done later.